rush.json

```diff
@@ -23,6 +23,12 @@
       "versionPolicyName": "lockedMajor",
       "shouldPublish": true
     },
+    {
+      "packageName": "@uifabric/foundation",
+      "projectFolder": "packages/foundation",
+      "versionPolicyName": "lockedMajor",
+      "shouldPublish": true
```

Comment:
Careful with this being shouldPublish: true. I think we have to manually upload a version before we can merge this PR

Reply:
I pulled out any changes using this package from other packages to help with publishing. @cliffkoh should I change this to false?